docs/testhelpers.rst

```diff
@@ -33,10 +35,25 @@ This test will now only run if the test suite is invoked as
 
 
 Reference/API
-=============
+-------------
 
 .. module:: astropy.tests.helper
 
 .. automodapi:: astropy.tests.helper
     :no-main-docstr:
     :no-inheritance-diagram:
+
+
+Astropy Test Runner
+===================
+
+When executing tests with either `astropy.test` or ``python setup.py test`` the call to pytest is controlled by the `astropy.tests.runner.TestRunner` class.
```

Comment:
Can you add a few sentences explaining the changes in this PR and maybe giving a very simple example of how to write your own test keyword?  (even if it just produces hello_world).